packages/desktop_webview_window/lib/src/webview.dart

```diff
@@ -53,6 +53,9 @@ abstract class Webview {
   /// Navigate to the next page in the history.
   Future<void> forward();
 
+  /// Show or hide webview window
+  Future<void> showWebviewWindow(bool show);
```

Comment:
The method name showWebviewWindow seems a little bit wrong, because it also contains hide logic.

I prefer it to be more semantic. 
For example: 
1. Provide two methods `showXXX` and `hideXXX` 
2. change to `setWebviewWindowVisibility(bool visible)`

Reply:
Done. I did the 2nd option